frameworks/cassandra/docs/service-settings.md

```diff
@@ -43,4 +43,41 @@ You can configure whether the creation, transfer, and restoration of backups occ
 *   **DC/OS web interface**: `BACKUP_RESTORE_STRATEGY`: `string`
 
 # Virtual networks
+
 The Cassandra service can be run on a virtual network such as the DC/OS overlay network, affording each node its own IP address (IP per container). For details about virtual networks on DC/OS see the [documentation](/latest/networking/virtual-networks/#virtual-network-service-dns). For the Cassandra service, using a virtual network means that nodes no longer use reserved port resources on the Mesos agents.  This allows nodes to share machines with other applications that may need to use the same ports that Cassandra does. That means, however, that we cannot guarantee that the ports on the agents containing the reserved resources for Cassandra will be available, therefore we do not allow a service to change from a virtual network to the host network. **Once the service is deployed on a virtual network it must remain on that virtual network**. The only way to move your data to Cassandra on the host network is through a migration.
+
+# TLS
+
+It is possible to launch the Cassandra service over a TLS encrypted connection. Enabling TLS will switch Cassandra inter-node communication to TLS connections between all the Cassandra nodes. Cassandra will be still available on the same configured client port (default: `9042`).
```

Comment:
The second sentence is a bit long and could be reformulated: What about "Enabling TLS will use TLS connections for inter-node communication between all the Cassandra nodes."